api/src/shared/mail/infrastructure/services/mailer.js

```diff
@@ -94,6 +94,9 @@ class Mailer {
   get acquiredCleaResultTemplateId() {
     return mailing[this._providerName].templates.acquiredCleaResultTemplateId;
   }
+  get targetProfileNotCertifiableTemplateId() {
```

Comment:
Pinaillage : pour rester uniforme sur le fichier

```suggestion
  
  get targetProfileNotCertifiableTemplateId() {
```